pkg/controller/operators/olm/operator_test.go

```diff
@@ -3578,7 +3578,7 @@ func TestSyncOperatorGroups(t *testing.T) {
 
 			operatorGroup, err := op.GetClient().OperatorsV1alpha2().OperatorGroups(tt.initial.operatorGroup.GetNamespace()).Get(tt.initial.operatorGroup.GetName(), metav1.GetOptions{})
 			require.NoError(t, err)
-			assert.Equal(t, tt.expectedStatus, operatorGroup.Status)
+			assert.EqualValues(t, tt.expectedStatus, operatorGroup.Status)
```

Comment:
I know this is annoying to fix, but doesn't this mask the difference between "" vs nil?

Reply:
Doesn't look like  it? https://github.com/stretchr/testify/blob/master/assert/assertions.go#L78:6